src/napari/layers/labels/_labels_key_bindings.py

```diff
@@ -87,7 +87,12 @@ def new_label(layer: Labels):
                 )
             )
         else:
-            layer.selected_label = new_selected_label
+            try:
+                layer.selected_label = new_selected_label
+            except WrongSelectedLabelError as e:
+                show_warning(
+                    f'{e.text}\nYou may convert the layer dtype in right click menu on layer list.'
```

Comment:
```suggestion
                    f'{e.text}\nYou can convert the layer dtype in right-click menu of the layer list.'
```

Reply:
Should I move this text to same global variable?

Reply:
> Should I move this text to same global variable?

It might be easier to maintain? It might also apply to something else, like image layers?